index.html

```diff
@@ -72,27 +73,197 @@ <h2 class="intro_text">I am a Software developer. <br /></h2>
         </section>
       </div>
     </section>
-    <div class="second_section_div">
-      <div class="projects-card-one"></div>
-      <div><h3 class="recent_work">My Recent Works</h3></div>
+    <section class="second_section_container1" id="mobile_portfolio">
+      <div class="second_section_div">
+      <div class="recent_work">
+      <div><h3 class="recent_work">My Recent Works</h3>
+      <hr class="line_divider" />
+      </div>
       <div class="image_holder">
-        <!-- multiple stories image comes here -->
+         <img class="work-main-image" src="images/Img Placeholder.png.png" />
       </div>
       <div><h3 class="stories">Multi-Post Stories</h3></div>
-      <div>
+      <div class="work_description_">
         <p class="work_description">
           A daily selection of privately personalized reads; no accounts or
           sign-ups required. has been the industry's standard dummy text ever
           since the 1500s, when an unknown printer took a standard dummy text.
         </p>
       </div>
-      <div class="languages">
-        <li class="boxes">Html</li>
-        <li class="boxes">Css</li>
-        <li class="boxes">Ruby</li>
-        <li class="boxes">Javascript</li>
+      <ul class="languages">
+        <li>Html</li>
+        <li>Css</li>
+        <li>Bootstrap</li>
+        <li>Ruby</li>
+      </ul>
+      <div class="see_button">
+       <a class="button"> See Project </a>
+      </div>
+      </div>
+    </section>
```

Comment:
@Tadesse-Alemayehu,
Thank you for your review. but I'm curious when you say a single work container or section, what are you talking about? the specified code is already in a section hence, I need some clarity and taking into cognizance that I am a newbie to HTML/CSS, and the deadline is around the corner. I will appreciate all forms of clarity and assistance to avoid repeating the module. 
cheers.

Reply:
Dear @3nityGitHub, I have attached a video with more explanation of the requirement. I hope that helps. If not, don't hesitate to ask for more detail.